training/04_editing_eml.Rmd

```diff
@@ -108,3 +108,22 @@ qa_package(mn, resource_map_pid)
 * Validate, write and update your package using the respective functions.
 * Use the <a href = 'https://nceas.github.io/datateam-training/reference/final-checklist-before-notifying-submitter.html' target='_blank'>checklist</a> to review your submission.
 * Make edits where necessary, and publish updates as needed.
+
```

Comment:
This seems old and not consistent with our current policies. Can we just remove this entire section?

Reply:
@jeanetteclark Yeah since it's part of Exercise 3B, I can just take out that section!